src/nxdoc/nuxeo-server/installation/nuxeo-cluster-architecture/nuxeo-architecture-components/database-architecture.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Data Structure
+description:
+labels:
+    - deployment
+    - bchauvin
+    - architecture
+    - cluster
+    - content-review-lts2017
+    - lts2019-ok
+review:
+    comment: ''
+    date: '2021-03-10'
+    status: ok
+toc: true
+tree_item_index: 500
+---
+
+## Concept
+
+The database is a core component for your Nuxeo cluster infrastructure, since it will store the document hierarchy, all document properties, and will be used as well for various queries. Nuxeo supports many databases, both NoSQL and relational ones. Among them, MongoDB and PostgreSQL currently provide the best overall performances for Nuxeo usage.
```

Comment:
... for various queries. In addition to the document repository, the database is also used to persist directories and KeyValue service content.